README.md

```diff
@@ -121,9 +121,9 @@ To see the Java Agent in action with sample applications, see our
 
 ### Basic Configuration
 
-The service name resource attribute is the only configuration option
-that needs to be specified. You can set it by adding a `service.name`
-attribute as shown in the [example above](#getting-started).
+The service name is the only configuration option that is required. You can
+specify it by setting the `otel.service.name` system property as shown in the
+[example above](#getting-started).
```

Comment:
`service.name` is the only required setting. You can specify it by setting `otel.service.name` as in the [example above](#getting-started).

Reply:
(I believe I can't use the suggest changes feature of GitHub if I lack proper permissions.)